code/modules/mob/living/blood.dm

```diff
@@ -97,8 +97,14 @@
 		if(isturf(src.loc)) //Blood loss still happens in locker, floor stays clean
 			if(amt >= 10)
 				add_splatter_floor(src.loc)
+				playsound(loc, pick('sound/effects/blood_puddle_splash.ogg'), 30, 1)
```

Comment:
playsound(src, again, and in all other places not going to repeat.

You're also calling pick on a  single element here and below, no need for that.

Reply:
thanks